test/expect-tests/dune_init_proj/dune_init_proj_tests.ml

```diff
@@ -0,0 +1,46 @@
+open Stdune
```

Comment:
Sorry, I mentioned expect thest but this is better as a cram test, since it only calls commands, for which `cram` is way better suited.

Reply:
@gabyfle most of the `cram` test for this subcommand are in https://github.com/ocaml/dune/tree/main/test/blackbox-tests/test-cases/dune-init.t/run.t